packages/front-end/components/Features/Rule.tsx

```diff
@@ -214,36 +218,15 @@ export const Rule = forwardRef<HTMLDivElement, RuleProps>(
                 >
                   {rule.enabled ? "Disable" : "Enable"}
                 </Button>
-                {environments
-                  .filter((e) => e.id !== environment)
-                  .map((en) => (
```

Comment:
New orgs by default have a single `production` environment, so this button will be broken for a lot of users (not "broken" exactly, it just will open a modal that doesn't do anything).  Is it worth adding logic to only render this button if there is at least 1 other environment?